src/framework/piral-core/src/components/wrapComponent.tsx

```diff
@@ -31,15 +31,15 @@ function wrapForeignComponent<T>(
 ) {
   return React.memo((props: T) => {
     const { destroyPortal, navigation } = useGlobalStateContext();
-    const id = React.useMemo(() => (portalIdBase++).toString(26), none);
+    const id = React.useMemo(() => (portalIdBase++).toString(26), [props.children]);
```

Comment:
I don't see why a change of props.children should have an impact on the id of the portal.

Maybe let's first have a MWE to reproduce the issue and then let's see what the root cause is / how to improve it.

Reply:
The situation is a bit complicated, but it does affect the update of components, resulting in abnormal rendering. I will show you this MWE later.

Reply:
Sure - thanks for the efforts!

Reply:
Any update on this? 1.5.3 is out and 1.5.4 is in the making.

Reply:
Yep. It is in processing, I'm using the develop branch to make the MWE.

Reply:
I need more time to review the details of the project, so I will close it for now and will reopen it when there is progress. It's most likely not the `piral-core`'s problem, and sorry for this.

Reply:
All good - just reopen / open an issue if we have a reproducible or can pin it down to Piral in any way.